Server/src/routes/vehicle.route.ts

```diff
@@ -86,11 +89,13 @@ export class VehicleRoute {
 	 * @returns Nothing.
 	 */
 	private async updateVehicleApp(req: Request, res: Response): Promise<void> {
-		const userData: UpdateRequestApp = req.body
-		if (!userData) {
+		const userDataPayload = UpdateRequestApp.safeParse(req.body)
+		if (!userDataPayload.success) {
+			logger.error(userDataPayload.error)
 			res.sendStatus(400)
 			return
 		}
+		const userData = userDataPayload.data
 
 		const userVehicle: Vehicle | null = await database.vehicles.getById(userData.vehicleId)
```

Comment:
Following comment in line 107 can be removed: // TODO: validate before with zod, jsonschema, io-ts, ts-auto-guard

Reply:
Ay, removed the comment. Thanks!